frontend/src/composables/useShootContext/index.js

```diff
@@ -921,22 +984,49 @@ export function createShootContextComposable (options = {}) {
           id,
           type,
           secretName,
+          secretRefName: utils.getResourceRefName(initialManifest.value?.spec?.resources, secretName),
```

Comment:
inconsistent usage of `?` (`initialManifest.value?`), see also https://github.com/gardener/dashboard/blob/5b26d8b50377e0a0254c08d70726ffb714e60d95/frontend/src/composables/useShootContext/index.js#L975
https://github.com/gardener/dashboard/blob/5b26d8b50377e0a0254c08d70726ffb714e60d95/frontend/src/composables/useShootSpec.js#L135

Reply:
Yes it is inconsistent and we do the same thing twice. I would prefer to use lodash get if we are unsure about the fact if an object is nullish or not.